guide/how-it-works/custom-spending-conditions.md

```diff
@@ -0,0 +1,225 @@
+
+#### Powered by Miniscript
+
+Wallets that support these kinds of configurations typically rely on [Miniscript](/guide/glossary/#miniscript) to implement them. Miniscript is a simplified version of Bitcoin Script which helps to reduce the complexity of Bitcoin Script, the native programming language for Bitcoin. Miniscript is easier to read by developers, and also allows for various build-tools to help ensure that scripts are safe, valid, and efficient.
+
+## Practical example
+
+Imagine a multi-key wallet with a 2-of-3 signing threshold that should turn into a 1-of-3 after six months of inactivity. In order to have an additional safety layer, we also want the wallet to have a fourth key which should activate after one year.
+
+This wallet has three layers of spending conditions and uses relative timelocks to unlock layers two and three: 
+
+- **Condition 1 (primary):** The primary spending condition defines that two out of three keys are required to sign a transaction.
+- **Condition 2 (recovery):** After 6 months of wallet inactivity, unlock a second spending condition which only requires one of the three keys to sign transactions.
+- **Condition 3 (emergency):** After 12 months of wallet inactivity, unlock a third spending condition which activates a fourth key that is not part of the first two conditions.
+
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-schedule.png"
+   retina = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-schedule@2x.png"
+   mobile = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-schedule-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-schedule-mobile@2x.png"
+   alt-text = "An illustration showing the unlocking scheme of the spending conditions."
+   width = 1600
+   height = 800
+%}
+
+</div>
+
+
+One important thing to note is that, once unlocked, spending conditions always remain active even if additional ones get unlocked. In the above scenario, you can sign a transaction with two keys (condition 1) but also with any one key after six months (condition 2).
+
+## How it works
+
+### Wallet creation and setup
+
+Similar to regular multi-key wallets, a software wallet application or coordination software initiates a multi-key wallet, choosing the number of total keys, and the number of keys required to sign transactions (called threshold). You then add public keys from other wallets generated elsewhere to the multisig after which the software wallet can complete the creation process. Please refer to the [Multi-key](/guide/how-it-works/private-key-management/multi-key/) page for a more detailed overview about how multi-key wallets work.
```

Comment:
By `public keys`, do you mean [extended public keys](https://bitcoin.design/guide/glossary/#extended-public-key-xpub-ypub-zpub)? If so, you could add that in parentheses with a link to the glossary.